faq.md

```diff
@@ -62,101 +100,125 @@ Commander X16P Features & Specifications (subject to change)
+Compilers are available for several programming languages: at least two
+different C compilers support the X16 as a compiler target, with appropriate
+runtimes. We also have a FORTH compiler in progress, plus a new programming
+language named Prog8. You can also use a dialect of BASIC called XC=BASIC. Of
+course, 65C02 assembly language is also an option, and all of the popular 6502
+assemblers can output object code that works on the X16. 
 
 ## Will it be compatible with Commodore 64 software?
 
-No, this is not an emulation of any prior computer. The X16 is a unique product with its own memory layout and devices. It can run simple BASIC programs written for BASIC 2, and it can even run machine language programs that strictly use KERNAL calls for input and output. It will not, however, run Commodore games without re-writing the graphics, keyboard, joystick, and storage routines for this platform. 
+No, this is not an emulation of any prior computer. The X16 is a unique product
+with its own memory layout and devices. It can run simple BASIC programs written
+for BASIC 2, and it can even run machine language programs that strictly use
+KERNAL calls for input and output. It will not, however, run Commodore games
+without re-writing the graphics, keyboard, joystick, and storage routines for
+this platform. 
 
 ## Can this emulate <some other computer>?
 
-No. The 6502 does not have the necessary hardware for realtime emulation of other system architectures. Attempting to emulate any other computer in real time is going to be virtually impossible. Again, some things can be simulated at a superficial level, but full emulation that can run arbitrary machine code from another platform is unlikely.
+No. The 6502 does not have the necessary hardware for realtime emulation of
+other system architectures. Attempting to emulate any other computer in real
+time is going to be virtually impossible. Again, some things can be simulated at
+a superficial level, but full emulation that can run arbitrary machine code from
+another platform is unlikely.
 
 ## Why the name "X16"?
 
-Commodore already has a "C16" computer: the Commodore 16. Obviously, we don't want people confusing "C16" and "C16". So the "X" was added to differentiate our computer. Perifractic proposed the always cool "X" be added to make "X16", and this got the team's vote. 
+Commodore already has a "C16" computer: the Commodore 16. Obviously, we don't
+want people confusing "C16" and "C16". So the "X" was added to differentiate our
+computer. Perifractic proposed the always cool "X" be added to make "X16", and
+this got the team's vote. 
 
-The "16" part comes from the intention to use a 65C816 processor. While that plan was scrapped, due to the need for additional hardware to demux the address and data lines, the name has not changed. 
+The "16" part comes from the intention to use a 65C816 processor. While that
+plan was scrapped, due to the need for additional hardware to demux the address
```

Comment:
While you're here, perhaps this and the specs should be updated to reflect the current plans for 65816 to at least be an option? I'm not interested enough on the exact processor to keep up with things, but it seems to be the plan has been un-scrapped.

Reply:
This thought definitely crossed my mind too but I was trying to avoid the FAQ being a cause for contention. 816 caused some passionate debates. I actually started updating this as a means to help avoid the recent "spirited" post on the forums so I was wanting to be careful about either adding more questions or opening the unhelpful discussions.

I'm not opposed to adding a mention of work being done for the 816 though. At this stage I would be surprised if the 816 wasn't going to be end up being an official upgrade at some point though this hinges on some of the KERNAL/ROM changes and there's still work to be done there. The x16-docs clearly mention the X16 is still a 65C02 based computer as of today, though there is an article in the faq section that goes into details. We could cross link one or both perhaps.

If we include it, we should probably be a little hand waivey and use terms like "may" vs "will" as an example.